slackclient/client.py

```diff
@@ -136,6 +136,42 @@ def rtm_read(self):
         else:
             raise SlackNotConnected
 
+    def rtm_send_message(self, channel, message, thread=None, reply_broadcast=None):
+        '''
+        Sends a message to a given channel.
+
+        :Args:
+            channel (str) - the string identifier for a channel or channel name (e.g. 'C1234ABC',
+            'bot-test' or '#bot-test')
+            message (message) - the string you'd like to send to the channel
+            thread (str or None) - the parent message ID, if sending to a
+                thread
+            reply_broadcast (bool) - if messaging a thread, whether to
+                also send the message back to the channel
+
+        :Returns:
+            None
+
+        '''
+        found_channel = self.server.channels.find(channel)
+        if found_channel:
+            # If the channel is found in cache, send to it's `send_message` method
+            return found_channel.send_message(
```

Comment:
why call this method on the channel if its just going to bounce out to server? i think you can consolidate some code if you just always call the method on server, just use `found_channel.id` as the argument.

Reply:
Good point. I'll update it.